BREAKING-CHANGES.md

```diff
@@ -58,6 +59,29 @@ This is a quick summary of the sections below:
   - MSRV is now 1.63.0
   - `List` no longer ignores empty strings
 
+## v0.28.0 (unreleased)
+
+### `Layout::init_cache` no longer returns bool and takes a `NonZeroUsize` instead of `usize` ([#1145])
+
+[#1145]: https://github.com/ratatui-org/ratatui/pull/1145
+
+```diff
+- let is_initialized = Layout::init_cache(100);
++ Layout::init_cache(NonZeroUsize::new(100).unwrap());
+```
+
+### `ratatui::terminal` module is now private ([#1160])
+
+[#1160]: https://github.com/ratatui-org/ratatui/pull/1160
+
+The `terminal` module is now private and can not be used directly. The types under this module are
+exported from the root of the crate.
```

Comment:
I would add the reason for the why here which is also given on the PR description. It's easier to adopt change when there is a helpful reason.

Reply:
> Also, I'm inclined to remove the terminal module as it reduces clashes with crossterm::terminal / termwiz::terminal, which are sometimes useful to reference as their module names.
> 
> from https://github.com/ratatui-org/ratatui/issues/1210#issuecomment-2264636585

Reply:
2e400f892